src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala

```diff
@@ -66,24 +68,33 @@ class HeartbeatActor(config: Heartbeat.Config) extends LoggingFSM[HeartbeatInter
 object Heartbeat {
   import org.slf4j.LoggerFactory
 
+  /** avoid log files that are overly verbose by only logging missed heartbeats after the first */
+  val LOG_AFTER_N_MISSES = 1
```

Comment:
Making this configurable gives the impression that we're controlling how many skipped heartbeats are OK before we consider it a problem.
I think the purpose of this patch is to distinguish between a potential incoming heartbeat, and an explicit one. The reason we don't want to log (or at least log something else) on the initial heartbeat is because we have no control over whether it will happen or not. This means it doesn't imply anything about the stability of the connection.

Our goal should be to log information that allows us to detect partitions or potential partitions.